e2e/journeys/citizen/createCase/C100.ts

```diff
@@ -64,15 +96,34 @@ export class C100 {
           urgencyAndWithoutNoticeAllOptionsYes,
       });
       // People
-      await C100Confidentiality.c100Confidentiality({
+      // await C100Confidentiality.c100Confidentiality({
+      //   page: page,
+      //   accessibilityTest: accessibilityTest,
+      //   errorMessaging: errorMessaging,
+      //   c100OthersKnowApplicantsContact: c100OthersKnowApplicantsContact,
+      //   c100PrivateDetails: c100PrivateDetails,
+      // });
```

Comment:
is this meant to be commented out?

Reply:
Yeah this is needed as the prior pages aren't ready yet